oteltest/tracer.go

```diff
@@ -31,7 +31,8 @@ type Tracer struct {
 	// Version is the instrumentation version.
 	Version string
 
-	config *config
+	config   *config
```

Comment:
nit: we could remove the `config` field as it is a duplicate of the `provider.config` field.